### PR TITLE
Remove balance check

### PR DESCRIPTION
**Pull request by rotcivegaf**

### CODE REVIEW

File: contracts/libs/SafeERC20.sol [hunk 1]
```diff
@@ -57,17 +52,11 @@ library SafeERC20 {
     function safeTransferFrom(
         IERC20 _token,
         address _from,
-        address _to, 
+        address _to,
         uint256 _value
-    ) internal returns (bool) 
-    {
+    ) internal returns (bool) {
         uint256 prevBalance = _token.balanceOf(_from);
```
Comment (nachomazzara) [inline contracts/libs/SafeERC20.sol:60]: Why do you want to remove it?

Comment (rotcivegaf) [reply contracts/libs/SafeERC20.sol]: Sorry, i used replace all

Comment (rotcivegaf) [reply contracts/libs/SafeERC20.sol:60]: You save a line but you're right. I reverse this change

**Comment by nachomazzara** [comment]
Can we close this PR in favor of #4?

**Pull request closed without merging**